web/pandas/pdeps/0008-inplace-methods-in-pandas.md

```diff
@@ -290,12 +290,13 @@ lazy copy" to be triggered under Copy-on-Write. It is also hard to fix, adding a
 with `inplace=True` might actually be worse than triggering the copy under the hood. We would only copy columns that
 share data with another object, not the whole object like `.copy()` would.
 
-There is another possible variant, which would be to trigger the copy (like the first option), but have an option to
-raise a warning whenever this happens.
+**Therefore, we propose to silently copy when needed.** The `inplace=True` option would thus mean "try inplace whenever possible", and not guarantee it is actually done inplace.
+
+In the future, if there is demand for it, it could still be possible to add to option to raise a warning whenever this happens.
 This would be useful in an IPython shell/Jupyter Notebook setting, where the user would have the opportunity to delete
 unused references that are causing the copying to be triggered.
 
-For example,
+For example:
 
     :::ipython
     In [1]: import pandas as pd
```

Comment:
Do we want to keep this example? It's a quite long example (and the text is already long) and I am not sure if it is critical for the explanation. It essentially shows that with IPython, an inplace setitem actually triggers CoW because of IPython storing the variable when displayed. 
But that's also something we can mention in words as a typical example where you might unexpectedly get silent copies with `inplace=True`

Reply:
Not opposed to removing it, but it might be better to put it under a collapsible block.

Reply:
Yeah, that would be nice, but the problem is that our current "rendering" step of the web pages is rather basic, and it doesn't work with html `<details>`.

Reply:
Will leave this as is for now, and look a bit into better rendering again.